src/calculation/DetailedResultsBuilder.ts

```diff
@@ -37,29 +37,45 @@ export function createPopulationValues(
     // collect results per episode
     episodeResults = createEpisodePopulationValues(populationGroup, patientResults);
 
-    // if no episodes were found we need default populationResults/stratifier results. Just use the patient
+    // if no episodes were found we need default populationResults/stratifierResults. just use the patient
```

Comment:
```suggestion
    // if no episodes were found we need default populationResults/stratifierResults. Just use the patient
```
This is the most minor suggestion ever so I'll just leave it as a comment if you feel like changing it haha

Reply:
Haha dang I actually changed this over in #163 but we must've axed it by accident during the rebase. Bring it back!!